node_modules/gh-orgunit/lib/api.js

```diff
@@ -508,6 +508,16 @@ var addOrgUnitSeries = module.exports.addOrgUnitSeries = function(ctx, id, serie
                 return callback({'code': 400, 'msg': 'Some of the provided series belong to another application'});
             }
 
+            // Get all the series that are NOT part of the organisational unit yet
+            var newSeries = _.filter(series, function(serie) {
+                return !_.findWhere(orgUnit.Series, {'id': serie.id});
+            });
+
+            // If there are no new series, we can return early
+            if (_.isEmpty(newSeries)) {
```

Comment:
Yes. Originally, I was going to do an if (err && err.name == UniqueIndexViolation) .. thing after the add, but it might get confusing when you're trying to add series to org units and only 1 is a duplicate